pep-0001.txt

```diff
@@ -668,7 +670,9 @@ handled through GitHub issues and pull requests, but you may also use
 
 For each new PEP that comes in an editor does the following:
 
-* Make sure a core developer is either an author or a sponsor of the PEP.
+* Make sure a core developer is either an author or a sponsor of the PEP has
+  been selected (non-core developer sponsors must be approved by the Steering
+  Council).
```

Comment:
"Make sure that the PEP is either co-authored by a core developer, has a core developer as a sponsor, or has a sponsor specifically approved for this PEP by the Steering Council."